### PR TITLE
Add new init method

### PR DESCRIPTION
**Pull request by ThulasikaV**

### CODE REVIEW

File: googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml [hunk 1]
```diff
@@ -17,30 +17,64 @@
...
+        <property name="uri.var.refreshToken" expression="$func:refreshToken"/>
+        <property name="uri.var.clientId" expression="$func:clientId"/>
+        <property name="uri.var.clientSecret" expression="$func:clientSecret"/>
+        <property name="uri.var.apiUrl" expression="$func:apiUrl"/>
+        <property name="uri.var.accessToken" expression="$func:accessToken"/>
+        <property name="uri.var.accessTokenRegistryPath" expression="$func:accessTokenRegistryPath"/>
         <property name="uri.var.apiVersion" expression="$func:apiVersion"/>
         <property name="uri.var.appVersionId" value=""/>
-
         <filter xpath="(not(get-property('uri.var.apiVersion') = '' or (not(string(get-property('uri.var.apiVersion'))))))">
```
Comment (malakasilva) [inline googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml]: Replace get-property with ctx.

File: googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml [hunk 2]
```diff
@@ -17,30 +17,64 @@
...
+            <then>
+                <property name="uri.var.accessTokenRegistryPath"
+                          value="connectors/Google/accessToken"/>
+            </then>
+            <else>
+                <property name="uri.var.accessTokenRegistryPath"
+                          expression="$ctx:uri.var.accessTokenRegistryPath"/>
+            </else>
+        </filter>
+        <filter source="boolean($ctx:uri.var.accessToken)" regex="false">
```
Comment (malakasilva) [inline googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml]: Change to following please,

<property name="uri.var.accessToken"
                          expression="get-property('registry', $ctx:uri.var.accessTokenRegistryPath)"/>
        <filter source="boolean($ctx:uri.var.accessToken)" regex="false">
               <then>
                     <property name="uri.var.accessToken" expression="$func:accessToken"/>
               </then>
        </filter>

File: googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/getAccessTokenFromRefreshToken.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (malakasilva) [inline googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/getAccessTokenFromRefreshToken.xml]: Year needs o be 2016

File: googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml (hunk 2 above)
Comment (ThulasikaV) [inline googlespreadsheet/googlespreadsheet-connector/googlespreadsheet-connector-2.0.0/org.wso2.carbon.connector/src/main/resources/config/init.xml]: $ctx function is not working for getting value from registry.

**Pull request merged**